packages/now-static-build/src/frameworks.ts

```diff
@@ -379,7 +379,17 @@ const frameworkList: Framework[] = [
     slug: 'nuxtjs',
     dependency: 'nuxt',
     buildCommand: 'nuxt generate',
-    getOutputDirName: async () => 'dist',
+    getOutputDirName: async (dirPrefix: string) => {
+      if (await isDir(join(dirPrefix, 'dist'))) {
+        return 'dist';
+      }
+
+      if (await isDir(join(dirPrefix, 'public'))) {
+        return 'public';
+      }
+
+      return 'dist';
+    },
```

Comment:
Can you add a test for this new behavior?

Reply:
@styfle I don't think that's necessary, since it's more of a temporary adjustment until it's possible to specify the output directory for the project, and we already have tests for Nuxt.js in general.